pkg/registry/minion/rest.go

```diff
@@ -91,7 +91,18 @@ func (rs *REST) Get(ctx api.Context, id string) (runtime.Object, error) {
 
 // List satisfies the RESTStorage interface.
 func (rs *REST) List(ctx api.Context, label, field labels.Selector) (runtime.Object, error) {
-	return rs.registry.ListMinions(ctx)
+	nodes, err := rs.registry.ListMinions(ctx)
```

Comment:
Yes, but node/minion needs to be converted to use the generic etcd implementation, similar to pods:
https://github.com/GoogleCloudPlatform/kubernetes/blob/master/pkg/registry/generic/etcd/etcd.go#L138
```
func (r *REST) List(ctx api.Context, label, field labels.Selector) (runtime.Object, error) {
	return r.store.List(ctx, label, field)
}
```